### PR TITLE
I wrote a sentence

### PR DESCRIPTION
**Pull request by crazicus**
I did great job with it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 This is a demo repository to show members how github works
 
 Hello World!
+
+Hye lookk at my awesome englih
```
Comment (heorg) [inline README.md]: There are grammatical errors. Please fix, before you're fired.

**Review by heorg — COMMENTED** [review]
Please fix

**Review by crazicus — COMMENTED** [review]
I fixed it!

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 This is a demo repository to show members how github works
 
 Hello World!
+
+Hey look at my awesome English!
```
Comment (heorg) [inline README.md:6]: This looks great! I'll approve it!

**Review by heorg — COMMENTED** [review]
Great job!

**Pull request merged**